CONTRIBUTING.md

```diff
@@ -50,6 +50,8 @@ For example, `add/gallery-block` means you're working on adding a new gallery bl
 
 You can pick among all the <a href="https://github.com/WordPress/gutenberg/issues">tickets</a>, or some of the ones labelled <a href="https://github.com/WordPress/gutenberg/labels/Good%20First%20Issue">Good First Issue</a>.
 
+If you're interested, feel free to [read on about how the repository is managed](./docs/repository-management.md).
```

Comment:
The tone on this seems a bit offputting.  Maybe something like,

`The workflow is documented in greater detail in the [repository_management] document.`

Reply:
Good point. Updated in c0e24ab3